monai/networks/blocks/hyena.py

```
@@ -0,0 +1,548 @@
+
+    instead of the full ``(B × C × ...)`` allocation.
+    """
+
+    _spatial_dims: int  # set by subclasses
+    fft_chunk_size: int = 0  # 0 = no chunking; set in subclass __init__
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        spatial = x.shape[2:]
+        kernel_shape = self.weight.shape[2:]  # type: ignore[attr-defined]
+        fft_dims = tuple(range(-self._spatial_dims, 0))
+        fft_size = [s + k - 1 for s, k in zip(spatial, kernel_shape)]
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)